data-prepper-plugins/s3-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/configuration/ObjectKeyOptions.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.sink.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * An implementation class of path prefix and file pattern configuration Options
+ */
+public class ObjectKeyOptions {
+    private static final String DEFAULT_FILE_PATTERN = "events-%{yyyy-MM-dd'T'hh-mm-ss}";

```

Comment:
Should this be `events-%{yyyy-MM-dd'T'hh-mm-ss}-%{epochSeconds}-%{random}.%{extension}`.

Either that, or we should rename the property to be clearer that this is only part of the name.

Reply:
Modifid the property name from `DEFAULT_FILE_PATTERN` to `DEFAULT_NAME_PATTERN`.